### PR TITLE
Add regenerate-keys option to Issue command to always regenerate key pair

### PR DESCRIPTION
**Pull request by Ayesh**
Hallo Niklas,
Thanks for this great acme client. I use it almost every day in many servers I manage, and it's working quite well!

Currently, `acme-client` reuses existing private key if one is present in the pointed storage directory. This is one difference between EFF's `certbot`; By default, `certbot` always regenerates a new private key on each renewal. With `acme-client`, one has to delete the existing private key or point to a different directory to make `acme-client` generate a new key. None of these options are convenient as a command line option. 

I added one, and it seems to work OK at my end. I'm sending a PR hoping you could take a look, and perhaps merge if you think it suits. 

Thank you 👍🎉

### CODE REVIEW

**Comment by kelunik** [comment]
Looks pretty fine. Do you want to add a corresponding option to `auto` as well?

File: src/Commands/Issue.php [hunk 1]
```diff
@@ -109,11 +109,17 @@ private function doExecute(Manager $args) {
         $path = "certs/" . $keyFile . "/" . reset($domains) . "/key.pem";
         $bits = $args->get("bits");
 
-        try {
+        if ($args->get("regenerate-keys")) {
+          $keyPair = (new OpenSSLKeyGenerator)->generate($bits);
+          $keyPair = (yield $keyStore->put($path, $keyPair));
+        }
+        else {
```
Comment (kelunik) [inline src/Commands/Issue.php]: Please move that to the previous line (`} else {`) and indent the code with four spaces.

**Comment by Ayesh** [comment]
Thanks for replying! I have force-pushed that commit with code styling changes. I'm sorry for not paying attention to code styling. 

I'll also work on changes in the `auto` command.

**Comment by kelunik** [comment]
@Ayesh Do you still plan to work on `auto`?

**Comment by Ayesh** [comment]
I'm so sorry about the delay on my side. I have made some progress and will commit them later today. Thank you for the follow-up.

**Comment by kelunik** [comment]
@Ayesh Implemented it in https://github.com/kelunik/acme-client/commit/51acff5bd3216c60a86a110b687210693fb5c0af slightly differently. I deferred writing the new key until the certificate is successfully issued, so things won't break if issuance fails.

Thanks for your contribution.

**Pull request closed without merging**